### ISSUE TITLE
Examples of successful run for RL training?

### ISSUE THREAD

**Issue by Some-random** [willccbb/verifiers#32]
I'm impressed with reward curves that look like this 

![Image](https://github.com/user-attachments/assets/c1fd41d3-8508-411f-893c-4547628f5365)

But I'm not which task was it... I tried to run the starter [gsm8k task](https://github.com/willccbb/verifiers?tab=readme-ov-file#usage) on readme but the [loss](https://wandb.ai/dongwei_jiang/huggingface/runs/wtfybuqp) seems quite bumpy...

**Comment by willccbb** [comment]
this is a run of the math_calculator task. 

<img width="390" alt="Image" src="https://github.com/user-attachments/assets/4e5f4e06-6478-4504-a8c1-b01e2445fc69" />


The README task is overly simplified for demonstrative purposes, hyperparameters are probably not ideal. I haven't fully solidified what the best defaults should be, have leaned towards keeping ones that won't cause OOM on smaller GPUs in the `default_config` function, but overriding in the examples. `openbookqa_search` also has been pretty stable for me:

<img width="377" alt="Image" src="https://github.com/user-attachments/assets/7e92a36e-79ad-4202-81e4-05d0037ba1f0" />

More tasks/better defaults coming soon, hang tight. This is all "open research code" for now.

**Comment by Some-random** [comment]
Could you provide more details about the specific script you're running for math_calculator? I'm trying to understand which one you're using:

Is it [math_simple.py ](https://github.com/willccbb/verifiers/blob/fcedb6bc5285bb5ff6b29efed4a1ebe88977649c/verifiers/examples/math_simple.py)? I noticed this script doesn't appear to use any tools.

Or perhaps [gsm8k_calculator.py ](https://github.com/willccbb/verifiers/blob/fcedb6bc5285bb5ff6b29efed4a1ebe88977649c/verifiers/examples/gsm8k_calculator.py)? I attempted to run this one but encountered errors related to tensor placement and train batch size not being divisible.

Or is it another script entirely?

**Comment by Some-random** [comment]
@willccbb Sorry for pinging directly but not sure whether closed issues would still send notifications to you, could use some help here

**Comment by willccbb** [comment]
Sorry, yes, gsm8k_calculator. That issue usually indicated a version issue in your environment, please follow the setup instructions exactly.
